workspaces/orchestrator/app-config.yaml

```diff
@@ -81,7 +81,7 @@ auth:
 # permission:
 #   enabled: true
 #   rbac:
-#     policies-csv-file: ../../plugins/orchestrator/docs/rbac-policy.csv
+#     policies-csv-file: ./docs/rbac-policy.csv
```

Comment:
If a relative path is used then it is from the `rhdh-plugins/workspaces/orchestrator/packages/backend`.

I would keep the former value